README.md

```diff
@@ -40,32 +40,32 @@
  On this page display the names of files & folders (as determined by an API call) for the selected usernames’ repository.
  
  ## Task 7.
-  Stretch exercise.
+  //Stretch exercise.
   
-  Make a new route GET /github/users/:username/repos/:repoName/*path
+  //Make a new route GET /github/users/:username/repos/:repoName/*path
```

Comment:
Should we add a bit more guidance on how to break this large task up?

Reply:
i enjoyed that after the book one this one was more difficult with not as much guidance and i had to figure stuff out. I don't want to give too much information. What did you have in mind for guidance?